File: Order-Processing-Frontend/src/app/models/CartItem.ts
export class CartItem {
  isbn: number;
  title: string;
  price: number;
  publisherName: string;
  quantity: number;

  constructor() {
    this.isbn = 0;
    this.title = '';
    this.price = 0;
    this.publisherName = '';
    this.quantity = 1;
  }
}

File: Order-Processing-Backend/src/main/java/csed/database/orderprocessingbackend/model/cart/CartItem.java
package csed.database.orderprocessingbackend.model.cart;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class CartItem {
    private Long ISBN;
    private String title;
    private Double price;
    private String publisherName;
    private Integer quantity;
}
